apps/src/sketchlab/SketchlabView.tsx

```diff
@@ -175,6 +189,19 @@ const SketchlabView: React.FC<LabProps<LevelProperties>> = ({
           id="workspace"
           className={panelClassName}
           headerContent={<WorkspaceHeader />}
+          rightHeaderContent={
+            !readonlyWorkspace && (
```

Comment:
Is the readonly bit relevant to start over?

Reply:
We probably don't want to show the start over button if the workspace is read only?

Reply:
cc @dmcavoy re: readonly mode